pages/_app/index.jsx

```diff
@@ -21,7 +21,7 @@ process.on('uncaughtException', (err) => {
   Sentry.captureException(err)
 })
 
-const ROUTES_WITHOUT_STORE = ['/login']
+const ROUTES_WITHOUT_STORE = ['/login', '/reset']
```

Comment:
Buy the way, why do you need this? Is it bad if you pass store to the Component but do not use it there?

Reply:
it's not about the store. Pages specified in this constant don't require to fetch user at componentDidMount, i.e. it avoids redirecting to /login page when you are on /reset.

Reply:
We should handle it differently but not at the moment. Thank you for the explanation!